### PR TITLE
fix infinity wait

### PR DESCRIPTION
**Pull request by BetaRabbit**
If there is something wrong happened during the installation, node may wait forever.

### CODE REVIEW

File: installpresto.sh [hunk 1]
```diff
@@ -32,11 +32,19 @@ if [[ $(hostname -s) = hn* ]]; then
   wget https://repo1.maven.org/maven2/com/facebook/presto/presto-cli/$VERSION/presto-cli-$VERSION-executable.jar -O /usr/local/bin/presto-cli
   chmod +x /usr/local/bin/presto-cli
 
-  until slider registry  --name presto1 --getexp presto ; do
+  retry=0
+
+  until [[ slider registry  --name presto1 --getexp presto || retry -ge 60 ]]; do
     echo "waiting for presto to start.."
```
Comment (dharmeshkakadia) [inline installpresto.sh]: May be make message more useful, like 
"waiting for presto to start.. attempt $retry/60"

File: installpresto.sh [hunk 2]
```diff
@@ -32,11 +32,19 @@ if [[ $(hostname -s) = hn* ]]; then
...
+    let retry+=1
     sleep 10
   done
 
+  if [ "$retry" -ge 60 ]; then
```
Comment (dharmeshkakadia) [inline installpresto.sh]: Also, can you use "[[" instead of "[" for consistency ?

**Comment by BetaRabbit** [comment]
Hi @dharmeshkakadia , thanks for the review, I am an absolutely freshman in bash scripting, learned a lot from your comments. 😄
All changes are addressed, would you please help me review it again? Thanks!

**Comment by BetaRabbit** [comment]
Sorry, forgot to mention, I have tested it on Azure, it works as expected.

**Comment by dharmeshkakadia** [comment]
Thanks @BetaRabbit for your contribution !

**Pull request merged**